src/mudblazor/.template.config/template.json

```diff
@@ -52,13 +52,7 @@
         "**/wwwroot/bootstrap/**"
       ],
       "modifiers": [
-        {
-          "condition": "(!IndividualLocalAuth)",
-          "exclude": [
-            "**/wwwroot/bootstrap/**"
-          ]
-        },
-        {
```

Comment:
These lines ensure that bootstrap isn't included in MudBlazor templates.  However, we removed this folder in PR #465 and it won't appear again unless we start templates completely from scratch in the future.